test/helpers/simulated.go

```diff
@@ -65,10 +69,12 @@ func (t *TestClient) Call(result any, method string, args ...any) error {
 
 // SimulatedBackendSetup defines the setup for a simulated backend.
 type SimulatedBackendSetup struct {
-	UserAuth            *bind.TransactOpts
-	DeployerAuth        *bind.TransactOpts
-	BridgeProxyAddr     common.Address
-	BridgeProxyContract *polygonzkevmbridgev2.Polygonzkevmbridgev2
+	UserAuth                    *bind.TransactOpts
+	DeployerAuth                *bind.TransactOpts
+	BridgeProxyAddr             common.Address
+	BridgeProxyContract         *polygonzkevmbridgev2.Polygonzkevmbridgev2
+	AggOracleCommitteeAddr      common.Address
+	AggOracleCommitteeProxyAddr common.Address
```

Comment:
I don't see this belongs here, it should be related only for L2 setup/environment. Can we figure out if we can remove it as my suggestion is to calculate it in the `newSimulatedEVML2SovereignChain` function instead?

```suggestion
```

Reply:
done